test/scripts/streaming/run-all-tests.sh

```
@@ -0,0 +1,61 @@
+#!/bin/bash
```

Shellcheck (0.10.0)

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)